plexpy/config.py

```diff
@@ -297,6 +297,29 @@ def bool_int(value):
     'MONITOR_REMOTE_ACCESS': (int, 'Monitoring', 0),
     'MONITORING_INTERVAL': (int, 'Monitoring', 60),
     'MONITORING_USE_WEBSOCKET': (int, 'Monitoring', 0),
+    'MQTT_ENABLED': (int, 'MQTT', 0),
```

Comment:
`config.py` not used for notification agents in v2. No need to change this file.